### DISCUSSION TITLE
[@types/jasmine] Does Not Include `jasmine.is()` Even Though it has Existed Since `jasmine@3.10.0`

### DISCUSSION THREAD

**Discussion by SlurpTheo** [DefinitelyTyped/DefinitelyTyped#66604] [Issues with a @types package]
https://jasmine.github.io/api/edge/jasmine.html:
```
(static) is(sample)
Get an AsymmetricEqualityTester, usable in any matcher that passes if the actual value is the same as the sample as determined by the === operator.

Parameters:
Name	Type	Description
sample	Object	
The value to compare the actual to.
```

Can't find `is(` in https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/jasmine/index.d.ts

[a comment by an automated account is not shown]

**Comment by chivesrs** [comment]
You can send out a PR that adds this function.

Note that DefinitelyTyped is generally volunteer maintained so you won't always see changes from Jasmine edge immediately reflected in the typings.

**Comment by Semigradsky** [comment]
I see that it was added in 4.2.0 - https://github.com/jasmine/jasmine/blob/main/release_notes/4.2.0.md#new-features

**Comment by Semigradsky** [comment]
PR: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66924
